common/app/com/commercetools/sunrise/shoppingcart/CartComponent.java

```diff
@@ -70,16 +79,38 @@
 
     @Override
     public CartCreateCommand onCartCreateCommand(final CartCreateCommand cartCreateCommand) {
+        final CartDraft cartDraft = cartDraftWithAdditionalInfo(cartCreateCommand.getDraft());
```

Comment:
that example shows we could a wither for the drafts, you can create an issue if you agree https://github.com/commercetools/commercetools-jvm-sdk/issues/new